### PR TITLE
reafctor: Code cleanup

### PR DESCRIPTION
**Pull request by Xuanwo**
Signed-off-by: Xuanwo <github@xuanwo.io>

### CODE REVIEW

File: cmd/beyondctl/main.go [hunk 1]
```diff
@@ -38,3 +39,11 @@ func main() {
 		logger.Fatal("beyondctl execute", zap.Error(err))
 	}
 }
+
+func userConfigDir() string {
+	configDir, err := os.UserConfigDir()
```
Comment (Prnyself) [inline cmd/beyondctl/main.go:44]: Why not user home dir?

Comment (Xuanwo) [reply cmd/beyondctl/main.go:44]: Because we already have config dir here, we just need to follow the rule.

Comment (Xuanwo) [reply cmd/beyondctl/main.go:44]: Use home dir directly is not the best practice. In the Linux platform, we have `XDF_CONFIG_DIR` to set the user's config directory.

Comment (Xuanwo) [reply cmd/beyondctl/main.go:44]: See the comment of `os.UserConfigDir` for more details.

**Review by Prnyself — APPROVED** [review]
LGTM

**Pull request merged**